lib/include/my_wordarray.h

```diff
@@ -56,4 +56,8 @@ void my_wordarray_showln(char *const *array);
 **/
 void my_wordarray_show(char *const *array);
 
+int my_wordarray_size(char **wordarray);
```

Comment:
same as my_wordarray_len ?

Reply:
ah, i just read the function, add doc to not disapoint the reader please

Reply:
doc add